src/Compilers/CSharp/Test/Semantic/Semantics/IteratorTests.cs

```diff
@@ -339,5 +339,35 @@ public void TopLevelYieldBreak()
             Assert.NotNull(yieldNode);
             Assert.Equal(SyntaxKind.YieldBreakStatement, yieldNode.Kind());
         }
+
+        [Fact]
+        [WorkItem(11649, "https://github.com/dotnet/roslyn/issues/11649")]
+        public void IteratorRewriterShouldNotRewriteBaseMethodWrapperSymbol()
+        {
+            var text =
+@"using System.Collections.Generic;
+
+class Base
+{
+    protected virtual IEnumerable<int> M()
+    {
+        yield break;
+    }
+
+    class D : Base
+    {
+        protected override IEnumerable<int> M()
+        {
+            base.M(); // the rewriting of D.M() synthesizes a BaseMethodWrapperSymbol for this base call, with return type IEnumerable,
+                      // but it should not in turn be lowered by the IteratorRewriter
+            yield break;
```

Comment:
It may be worth testing the actual iteration functionality in addition to verifying that emit doesn't fail.